Tests/LibGfx/TestImageWriter.cpp

```diff
@@ -113,14 +113,23 @@ TEST_CASE(test_bmp)
 
 TEST_CASE(test_gif)
 {
-    // We only support grayscale and non-animated images yet
     auto bitmap = TRY_OR_FAIL(create_test_rgb_bitmap());
 
-    // Convert bitmap to grayscale
+    // We only support grayscale and non-animated images at the moment - convert bitmap to grayscale.
     for (auto& argb : *bitmap)
         argb = Color::from_argb(argb).to_grayscale().value();
 
-    TRY_OR_FAIL((test_roundtrip<Gfx::GIFWriter, Gfx::GIFImageDecoderPlugin>(bitmap, true)));
+    auto encoded_bitmap = TRY_OR_FAIL((encode_bitmap<Gfx::GIFWriter>(bitmap)));
+    auto decoder = TRY_OR_FAIL(Gfx::GIFImageDecoderPlugin::create(encoded_bitmap));
+
+    EXPECT_EQ(decoder->size(), bitmap->size());
+    EXPECT_EQ(decoder->frame_count(), 1u);
+    EXPECT(!decoder->is_animated());
+    EXPECT_EQ(decoder->loop_count(), 1u);
```

Comment:
Should we even test that?
Same for the duration, why enforcing this behavior in the decoder?

Reply:
Happy to remove whatever you want me to remove! Want to suggest an edit? Or just tell me which lines to remove.

Reply:
I was thinking of the two problematic lines, so `loop_count` and `duration`.